heartbeat/ocf-shellfuncs.in

```diff
@@ -1039,6 +1043,7 @@ ocf_attribute_target() {
 __ocf_set_defaults "$@"
 
 : ${OCF_TRACE_RA:=$OCF_RESKEY_trace_ra}
+: ${OCF_TRACE_DIR:=$OCF_RESKEY_trace_dir}
 ocf_is_true "$OCF_TRACE_RA" && ocf_start_trace
```

Comment:
> `trace_dir` is optional. The tracing works as before only if the `trace_ra` is set. (Which might look like that `trace_ra` is superfluous now).

That makes sense, it will be easy to leave trace_dir in there all the time, and just flip trace_ra on and off